ballerina/natives.bal

```diff
@@ -66,6 +67,18 @@ final map<int> & readonly logLevelWeight = {
     "DEBUG": 700
 };
 
+isolated string outputFilePath = "";
```

Comment:
use optional field instead of defaulting to empty string

Reply:
Updated in 98b4983